src/utils/junitUtils.ts

```diff
@@ -0,0 +1,76 @@
+import { SendJsonOutput } from '@/cli/send/jsonOutput';
```

Comment:
The import should be changed. I am afraid of an import loop with this. I would also definitely drag junitUtils over to cli, since this is only needed there.